src/components/ImageView/index.js

```diff
@@ -25,6 +25,8 @@ class ImageView extends PureComponent {
         this.onContainerPress = this.onContainerPress.bind(this);
         this.imageLoadingStart = this.imageLoadingStart.bind(this);
         this.imageLoadingEnd = this.imageLoadingEnd.bind(this);
+        this.onMouseUp = this.onMouseUp.bind(this);
```

Comment:
@mananjadhav could you suggest a better name?

Reply:
Maybe, mouseReleased?

Reply:
I just this comment. I posted a suggestion below. I'll try to explain the approach in writing method names.

Whenever we create a function, we should write their names based on the logic handled in the body and not related to the event.  So here for us it isn't relevant that we're attaching to the mouseup event. But the fact that the logic contains mouse pointer tracking or checking if the pointer is in the container.

Reply:
Thanks!